src/components/Button/Button.stories.tsx

```diff
@@ -213,25 +230,23 @@ LinkRightIcon.args = {
   variant: 'link',
 };
 
-export const Destructive = Template.bind({});
-Destructive.args = { children: 'Button', variant: 'destructive' };
-
-export const DestructiveDisabled = Template.bind({});
-DestructiveDisabled.args = {
+export const PrimaryAlert = Template.bind({});
```

Comment:
Same as "Tertiary" – I changed the story name from "Destructive" to "Primary Alert" so it's consistent with the props, but now it's at odds with the UI Kit. I can change it back if anyone feels strongly that it should match the UI Kit instead.